ai/ai_model_building/stat_parser.py

```diff
@@ -48,12 +43,33 @@ def numerise_field(value):
     return int(eval(value))
 
 
+def custom_load_unit_stat(stat, unit):
+    """Fetch the substats of a stat from a unit descriptor
+
+    Args:
+        stat (str): The stat's name
+        unit (dict): The unit descriptor
+
+    Returns:
+        dict: A dict with the substats values
+    """
+    result = {}
+
+    try:
+        for sub_stat in unit[stat]:
+            result[sub_stat] = [numerise_field(
+                unit[stat][sub_stat]) if sub_stat in unit[stat] else 0]
+
+        return result
+    except KeyError:
+        return {}
```

Comment:
What happend in this case ? Armor is C+1
<img width="669" alt="image" src="https://github.com/Prophecy-eip/Maths/assets/57897025/740b8c7b-5b6e-4695-8986-bc27760fb5cf">

Reply:
The special values like C+x are handle and converted into numeric values by the function "numerise_field" don't worry